.github/workflows/sycl_update_gpu_driver.yml

```diff
@@ -3,6 +3,11 @@ name: Update GPU driver
 on:
   schedule:
     - cron:  '0 3 * * 2'
+  pull_request:
+    branches:
+    - sycl
+    paths:
+    - .github/workflows/sycl_update_gpu_driver.yml
```

Comment:
This change enforce workflow when it's getting updated, basically it triggers workflow run in pre-commit. Like on the link you added. Otherwise pre-commit won't trigger this workflow at all.

Workflow itself by its purpose creates new PRs. The reason why last run (the one you referred to) was useless is that there've been no new driver version. I need to re-run pre-commit task (this workflow itself) as soon as new driver is available. That will create a completely new separate PR with driver version update, and I expect that will kick off standard pre-commit in new PR.

Reply:
I've updated sycl_precommit job to do not trigger pre-commit testing when sycl_update_gpu_driver is changed.
However in this specific PR it is still triggered ... because I changed sycl_precommit in this PR :)